### PR TITLE
[ NMA-988]: Default fiat currency

### PR DESCRIPTION
**Pull request by ClaudeHangui**
## Issue being fixed or feature implemented
Update the design of the screen which displays the list of supported fiat currencies
PS: There are no translations for the label **Select Currency**

## Checklist:

- [x] I have performed a self-review of my own code and added comments where necessary

### CODE REVIEW

File: wallet/src/de/schildbach/wallet/ui/ExchangeRatesFragment.java [hunk 1]
```diff
@@ -390,19 +399,31 @@ void setExchangeRates(List<de.schildbach.wallet.rates.ExchangeRate> exchangeRate
         }
     }
 
+    private Integer getFlagFromCurrencyCode(String currencyCode) {
+        if (currencyCode.equalsIgnoreCase("try")) return R.drawable.turk;    //we can't use 'try' as the recource name for turkish flag as try is a reserved keyword in java. A workaround is to use rename the turkish flag resource
```
Comment (tomasz-ludek) [inline wallet/src/de/schildbach/wallet/ui/ExchangeRatesFragment.java:403]: We should avoid such weird exceptions. I would suggest to add a prefix to each resource name (e.g. "currency_code_try", or something similar). Additionally, names with such prefix would be more self-descriptive.

**Review by tomasz-ludek — CHANGES_REQUESTED** [review]

File: wallet/src/de/schildbach/wallet/ui/ExchangeRatesFragment.java (hunk 1 above)
Comment (ClaudeHangui) [reply wallet/src/de/schildbach/wallet/ui/ExchangeRatesFragment.java]: The implementation of this method had been updated accordingly

**Review by ClaudeHangui — COMMENTED** [review]
- Our design for the exchange rates screen takes into account the 2 entry points we have: **Settings** and **SendPayment** flows. The action to display this new UI from the **SendPayment** flow has not yet been implemented
- We removed unused variable dependencies and refactored the class as much as we could

**Review by HashEngineering — APPROVED** [review]
looks good and works well

**Review by HashEngineering — CHANGES_REQUESTED** [review]
There is a merge conflict with the Firebase Events for Settings.

**Comment by ClaudeHangui** [comment]
Conflicts have been fixed

**Pull request merged**